### DISCUSSION TITLE
第5节课笔记(16班)

### DISCUSSION THREAD

**Discussion by Yang-Changhui** [InternLM/Tutorial#319] [笔记]
16班第5节课笔记提交在下面

**Comment by Eadral** [comment]
https://www.yuque.com/eadral/ruaugi/dvfhv8pzcme4alkg?singleDoc#

**Comment by morcake** [comment]
https://blog.51cto.com/morcake/9257044

**Comment by etheral12138** [comment]
https://y1yc0dwhb0o.feishu.cn/docx/C5TfdAHtVoLIUOxbm7YciTJlnPe?from=from_copylink

**Comment by 2404589803** [comment]
https://2404589803.github.io/My-datawhale-learning/#/shusheng_puyu_learnning53/ch05

**Comment by Dhaizei** [comment]
http://t.csdnimg.cn/Tlvcm

**Comment by hhhanqian** [comment]
http://xhslink.com/UQ4sLz

**Comment by Star-cre** [comment]
公开笔记：https://www.bilibili.com/video/BV1iW4y1A77P/?vd_source=2fb638751797274bd22bea982387a179
id：戈璧的老王

**Comment by Durian-1111** [comment]
**Day05-Homework**
链接：https://xby48gs45u.feishu.cn/docx/DrZIdynaGoHFy7xZfAxcfeCRnzh?from=from_copylink

**Comment by Aitejiu** [comment]
https://www.cnblogs.com/aitejiu/p/17975461

**Comment by zszbyzsz** [comment]
https://juejin.cn/post/7323867035636711465

**Comment by EnableAsync** [comment]
https://enableasync.github.io/internlm/internlm-05/
